ui/packages/catalog-ui-search/src/main/webapp/component/search-form-editor/search-form-editor.view.js

```diff
@@ -117,11 +121,48 @@ module.exports = Marionette.LayoutView.extend({
         showResultFilter: false,
         selectionInterface: new SelectionInterface(),
         onMapLoaded: olMap => {
+          const filterTemplate = this.model.get('filterTemplate')
           this.showQueryForm(collection, id)
+          if (!filterTemplate) {
+            return
+          }
+          const geoFilters = (
+            filterTemplate.filters || [filterTemplate]
+          ).filter(filter => CQLUtils.isGeoFilter(filter.type))
+          const coords = this.wktToCoords(geoFilters)
+          Common.queueExecution(() => {
+            this.map.currentView.map.zoomToExtent(coords)
+          })
         },
       })
     )
   },
+  wktToCoords(wktList) {
+    const parsedGeoCoords = wktList.map(wkt => {
+      if (wkt.value.startsWith('POINT') && parseFloat(wkt.distance) !== 0) {
+        const center = terraformer.parse(wkt.value).coordinates
+        return new TurfCircle(center, parseFloat(wkt.distance), 64, 'meters')
```

Comment:
❓ What is the significance of `64` here? Can we make it a constant with a descriptive name?

Reply:
it's the "number of steps", used for precision. the default in the new circle constructor is 64: https://turfjs.org/docs/#circle I'll make it a constant 👍